src/main/java/io/jenkins/plugins/pipelinegraphview/treescanner/NodeRelationshipFinder.java

```diff
@@ -57,14 +50,18 @@ public NodeRelationshipFinder() {}
     @NonNull
     public LinkedHashMap<String, NodeRelationship> getNodeRelationships(
             @NonNull LinkedHashMap<String, FlowNode> nodeMap) {
-        dump("Original Ids: %s", String.join(", ", nodeMap.keySet()));
+        if (isDebugEnabled) {
```

Comment:
could be simplified a bit if java.util.logging Logger was used and a supplier was passed to only create the string if debug is enabled, it would mean we could drop the `if debug is enabled`....

but no big deal.